src/app/interfaces/apiInterfaces.ts

```diff
@@ -41,6 +42,35 @@ export type DateTimeTimezone = DateTime;
 export function dateTimeTimezone(timestamp: string): DateTimeTimezone {
   return timestamp ? DateTime.fromISO(timestamp, { setZone: true }) : undefined;
 }
+/**
+ * Convert duration string into Duration
+ * @param seconds Duration seconds
+ */
+export function duration(seconds: number): Duration {
+  /*
+    Extra object fields required, do not remove. Duration calculates itself
+    based on the time spans provided, if years is removed for example,
+    the output will just keep incrementing months (i.e 24 months, instead of 2 years).
+  */
+  return seconds
+    ? Duration.fromObject({
+        years: 0,
+        months: 0,
+        days: 0,
+        hours: 0,
+        minutes: 0,
```

Comment:
```suggestion
```

Reply:
https://github.com/moment/luxon/issues/687. This is an intended feature, I must have those keys there.

Reply:
This is straight up counter intuitive. Should leave a comment behind at least (e.g. link to issue).